### ISSUE TITLE
Plaid is stuck on loading screen with spinner - Android only

### ISSUE THREAD

**Issue by osikes** [plaid/react-native-plaid-link-sdk#361]
# The problem

I have recently upgraded from  react-native-plaid-link-sdk 6.0.4 to 7.0.2. iOS works fine. But on Android, after I click the PlaidLink, the Plaid app pops up , but never gets past an initial white screen with a loading circle. 

# Environment

 Plaid Link React Native  7.0.2
 ReactNative Version 0.63.3                
Occurs on Android   yes
Android OS Version  version 8 and 9
 Android Devices/Emulators  motog6 play
 Occurs on iOS  no              

# Expected Result
I expect Plaid to finish loading with a list of banks to select from. Instead it stays on a loading screen.

# Screenshots
<img width="390" alt="Screen Shot 2021-04-14 at 9 48 50 AM" src="https://user-images.githubusercontent.com/955081/114740084-21ad2e00-9d0f-11eb-87cf-df8456fd10fa.png">

# Logs
Running logcat on device I see this show up when I click the plaid link

<img width="773" alt="Screen Shot 2021-04-14 at 10 52 28 AM" src="https://user-images.githubusercontent.com/955081/114740791-ca5b8d80-9d0f-11eb-92df-22199e928d1f.png">

The line I think is issue is '04-14 09:14:50.907 13965 14214 E WM-WorkerWrapper: java.util.concurrent.ExecutionException: java.lang.NoClassDefFoundError: Failed resolution of: Lpbandk/Message;'

# Code To Reproduce Issue 
 <PlaidLink
        tokenConfig={{ token: plaidToken }}
        products={['auth']}
        onExit={(response) => {
          alert(JSON.stringify(response))
        }}
        onSuccess={onSuccess}>
        <Button>
          <LinkYourBankText>Link Bank Account</LinkYourBankText>
        </Button>
      </PlaidLink>

**Comment by zsweigart** [comment]
Hi @osikes Thanks for the report.  In your top level `build.gradle` file inside the `android` directory would you be able to check that `jcenter()` is listed as a repository.  In our example app it is here

https://github.com/plaid/react-native-plaid-link-sdk/blob/master/example/android/build.gradle#L35

**Comment by osikes** [comment]
> Hi @osikes Thanks for the report. In your top level `build.gradle` file inside the `android` directory would you be able to check that `jcenter()` is listed as a repository. In our example app it is here
> 
> https://github.com/plaid/react-native-plaid-link-sdk/blob/master/example/android/build.gradle#L35

Yes, verified it is in those places @zsweigart

**Comment by sjbsiconnectguam** [comment]
Any updates on this?

**Comment by Crizzooo** [comment]
This is happening to us, but still on plaid-link-sdk 6.0.3.

In variant=debug, it seems to work fine, but for release variants, it just hangs.

Anybody know what's going on?

**Comment by Crizzooo** [comment]
Was this caused by upgrading to RN 63.3?

**Comment by amytang0** [comment]
Does this repro on 7.0.7?

**Comment by Crizzooo** [comment]
Hi @amytang0, thanks for reply!

This is desperately affecting a huge release for us here at https://www.titan.com, where we rely on Plaid for onboarding our clients. I really need to get this fixed by the end of this weekend.

We can try upgrading to 7.0.7. 

I'm working to try to uncover more about what's going on with our 6.0.3, and will have another developer trying to upgrade and repro in parallel.

As far as you know, Android works fine on that version?

**Do you have any insights as to what exactly could be failing when the Plaid window opens but infinitely spins?**  

I cant even find any logs that appear relevant when running `adb logcat` and really am not sure the path to debugging this.

**Comment by Crizzooo** [comment]
Unsure if any of these logs will help. This is output of `adb logcat | grep -C 3 "plaid"` immediately after initiating PlaidLink. Then it just hangs and no new logs appear

```
--
--
08-06 20:00:32.762 29292 29292 I ViewRootImpl@899d881[MainActivity]: ViewPostIme pointer 1
08-06 20:00:32.781  1267  3726 E ActivityTaskManager: intent received is not valid or null
08-06 20:00:32.781  1267  1386 D PackageManager: setEnabledSetting : userId = 0 packageName = com.titannative cmp = androidx.work.impl.background.systemalarm.RescheduleReceiver newState = 1 callingPackage = 10739/com.titannative
08-06 20:00:32.784  1267  3726 I ActivityTaskManager: START u0 {flg=0x4000000 cmp=com.titannative/com.plaid.link.LinkActivity} from uid 10739
08-06 20:00:32.792  1267  3726 D CustomFrequencyManagerService: acquireDVFSLockLocked : type : DVFS_MIN_LIMIT  frequency : 1766400  uid : 1000  pid : 1267  pkgName : AMS_ACT_START@CPU_MIN@23
08-06 20:00:32.793  1267  3726 D ActivityManagerPerformance: AMP_acquire() ACT_START
08-06 20:00:32.793  1267  3726 D ActivityTaskManager: MultiTaskingTaskLaunchParamsModifier:task=null display-from-source=0 display-id=0 display-windowing-mode=1
--
--
08-06 20:00:32.793  1267  3726 D ActivityTaskManager: MultiTaskingTaskLaunchParamsModifier:task=null display-from-source=0 display-id=0 display-windowing-mode=1
08-06 20:00:32.793  1267  3726 D ActivityTaskManager: MultiTaskingTaskLaunchParamsModifier:task=null display-from-source=0 display-id=0 display-windowing-mode=1 activity-options-fullscreen=Rect(0, 0 - 0, 0) non-freeform-display maximized-bounds
08-06 20:00:32.793  1267  3726 D ActivityTaskManager: updateMinimizedState: unknown notifyReason=2
08-06 20:00:32.795   914   999 I SurfaceFlinger: id=45262 createSurf (0x0),-1 flag=80004, AppWindowToken{2ac4cd7 token=Token{e5a5456 ActivityRecord{6212418 u0 com.titannative/com.plaid.link.LinkActivity t29895}}}#0
08-06 20:00:32.797  1267  3726 V WindowManager: Prepare app transition: transit=TRANSIT_ACTIVITY_OPEN mNextAppTransition=TRANSIT_UNSET alwaysKeepCurrent=true displayId=0 Callers=com.android.server.wm.DisplayContent.prepareAppTransition:6356 com.android.server.wm.DisplayContent.prepareAppTransition:6350 com.android.server.wm.ActivityStack.startActivityLocked:3999 com.android.server.wm.ActivityStarter.startActivityUnchecked:2924 com.android.server.wm.ActivityStarter.startActivity:2180
08-06 20:00:32.806   914   914 I Layer   : id=45263[0] Destroyed Screenshot Parent
08-06 20:00:32.814  1267  3726 D SensorService: removeConnection, index : 2, size : 5
--
--
08-06 20:00:32.852 29292 29292 D MultiWindowDecorSupport: setCaptionType = 0, DecorView = DecorView@c8825d6[]
08-06 20:00:32.858  1267  1490 D GameManagerService: noteResumeComponent(), received resumed-component: com.titannative
08-06 20:00:32.858 29292 26579 I System.out: (HTTPLog)-Static: isSBSettingEnabled false
08-06 20:00:32.859  1267  2036 D GameManagerService: onLooperPrepared(), msg: { when=-1ms what=10000 obj=ComponentInfo{com.titannative/com.plaid.link.LinkActivity} target=com.samsung.android.game.GameManagerService$FgCheckThread$1 }
08-06 20:00:32.859  1267  2036 D GameManagerService: sendPauseConditionally() - mPrevNotiPackage: com.sec.android.app.clockpackage
08-06 20:00:32.859  1267  2036 D GameManagerService: notifyResumePause(). of pkg: com.titannative, type: 4, isMinimized: false, isTunableApp: false
08-06 20:00:32.859  1267  2036 D GameManagerService:   notifyResumePause(). do nothing. mKillNotiCount: 1
--
--
08-06 20:00:32.883  1267  1386 E WindowManager: App trying to use insecure INPUT_FEATURE_NO_INPUT_CHANNEL flag. Ignoring
08-06 20:00:32.883  1267  1386 D InputTransport: Input channel constructed: '1e4c9eb', fd=275
08-06 20:00:32.883  1267  1386 D InputTransport: Input channel constructed: '1e4c9eb', fd=906
08-06 20:00:32.884   914  1099 I SurfaceFlinger: id=45264 createSurf (0x0),-1 flag=80004, 1e4c9eb com.titannative/com.plaid.link.LinkActivity#0
08-06 20:00:32.885  1267  1386 D InputTransport: Input channel destroyed: '1e4c9eb', fd=906
08-06 20:00:32.886 29292 29292 I ViewRootImpl@bc94799[LinkActivity]: setView = com.android.internal.policy.DecorView@c8825d6 TM=true MM=false
08-06 20:00:32.889 29292 29292 I ViewRootImpl@899d881[MainActivity]: MSG_WINDOW_FOCUS_CHANGED 0 1
--
--
08-06 20:00:32.889 29292 29292 D InputMethodManager: prepareNavigationBarInfo() DecorView@c55583d[MainActivity]
08-06 20:00:32.889 29292 29292 D InputMethodManager: getNavigationBarColor() -14735316
08-06 20:00:32.891  1267  4769 E ActivityTaskManager: intent received is not valid or null
08-06 20:00:32.893  1267  4769 I ActivityTaskManager: START u0 {flg=0x6000000 cmp=com.titannative/com.plaid.internal.link.LinkWebViewActivity} from uid 10739
08-06 20:00:32.893 29292 28656 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=f2912a2e-9d91-4975-8496-c73c5f832e17, tags={ com.plaid.internal.core.analytics.batch.AnalyticsBatchUploadWorker } ]
08-06 20:00:32.894  1267  4769 D ActivityTaskManager: MultiTaskingTaskLaunchParamsModifier:task=null display-from-source=0 display-id=0 display-windowing-mode=1
08-06 20:00:32.894  1267  4769 D ActivityTaskManager: MultiTaskingTaskLaunchParamsModifier:task=null display-from-source=0 display-id=0 display-windowing-mode=1 activity-options-fullscreen=Rect(0, 0 - 0, 0) non-freeform-display maximized-bounds
08-06 20:00:32.894  1267  4769 D ActivityTaskManager: updateMinimizedState: unknown notifyReason=2
--
--
08-06 20:00:32.894  1267  4769 D ActivityTaskManager: MultiTaskingTaskLaunchParamsModifier:task=null display-from-source=0 display-id=0 display-windowing-mode=1
08-06 20:00:32.894  1267  4769 D ActivityTaskManager: MultiTaskingTaskLaunchParamsModifier:task=null display-from-source=0 display-id=0 display-windowing-mode=1 activity-options-fullscreen=Rect(0, 0 - 0, 0) non-freeform-display maximized-bounds
08-06 20:00:32.894  1267  4769 D ActivityTaskManager: updateMinimizedState: unknown notifyReason=2
08-06 20:00:32.895   914  1099 I SurfaceFlinger: id=45265 createSurf (0x0),-1 flag=80004, AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}}#0
08-06 20:00:32.897  1267  4769 V WindowManager: Prepare app transition: transit=TRANSIT_ACTIVITY_OPEN mNextAppTransition=TRANSIT_ACTIVITY_OPEN alwaysKeepCurrent=true displayId=0 Callers=com.android.server.wm.DisplayContent.prepareAppTransition:6356 com.android.server.wm.DisplayContent.prepareAppTransition:6350 com.android.server.wm.ActivityStack.startActivityLocked:3999 com.android.server.wm.ActivityStarter.startActivityUnchecked:2924 com.android.server.wm.ActivityStarter.startActivity:2180
08-06 20:00:32.903  1267  1386 D PackageManager: setEnabledSetting : userId = 0 packageName = com.titannative cmp = androidx.work.impl.background.systemalarm.RescheduleReceiver newState = 2 callingPackage = 10739/com.titannative
08-06 20:00:32.905  1267  3765 I ActivityTaskManager: Remove from unknown app
--
--
08-06 20:00:32.903  1267  1386 D PackageManager: setEnabledSetting : userId = 0 packageName = com.titannative cmp = androidx.work.impl.background.systemalarm.RescheduleReceiver newState = 2 callingPackage = 10739/com.titannative
08-06 20:00:32.905  1267  3765 I ActivityTaskManager: Remove from unknown app
08-06 20:00:32.908  1267  4769 D SamsungAlarmManager: Cancel Alarm calling from uid:10739 pid :29292 / op:PendingIntent{c438592: PendingIntentRecord{51ee0d0 com.titannative broadcastIntent}}
08-06 20:00:32.908  1267  1386 V WindowManager: Relayout Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity}: viewVisibility=0 req=1080x2220
08-06 20:00:32.908 29292 26679 I Braze v13.1.1 .bo.app.i1: Completed the openSession call. Starting or continuing session fc2e608f-1408-4ed8-bc0c-0344364f2b6f
--
08-06 20:00:32.908 29292 26679 I Braze v13.1.1 .bo.app.i1: Completed the openSession call. Starting or continuing session fc2e608f-1408-4ed8-bc0c-0344364f2b6f
08-06 20:00:32.913   914  1099 I SurfaceFlinger: id=45266 createSurf (1080x2220),1 flag=404, com.titannative/com.plaid.link.LinkActivity$_29292#0
08-06 20:00:32.914  1267  1386 D WindowManager: makeSurface duration=1 name=com.titannative/com.plaid.link.LinkActivity$_29292
08-06 20:00:32.915  1267  1386 V WindowManager: Changing focus from null to Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:204 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6458 com.android.server.wm.WindowManagerService.relayoutWindow:2618 com.android.server.wm.Session.relayout:202 android.view.IWindowSession$Stub.onTransact:716 com.android.server.wm.Session.onTransact:142
08-06 20:00:32.915  1267  1386 D StatusBarManagerService: notifyRequestedSystemKey recent=false home=false
--
08-06 20:00:32.915  1267  1386 D StatusBarManagerService: notifyRequestedSystemKey recent=false home=false
08-06 20:00:32.915  1267  1488 D WindowManager: setSystemUiVisibility: displayId=0, vis=0xa008, fullVis=0x2000, dockVis=0x0, win=Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity}
08-06 20:00:32.916  1267  1386 V WindowManager: Relayout 29292: mAttrs={(0,0)(fillxfill) sim={adjust=pan forwardNavigation} ty=BASE_APPLICATION wanim=0x1030305
08-06 20:00:32.916  1267  1386 V WindowManager:   fl=81810100
08-06 20:00:32.916  1267  1386 V WindowManager:   pfl=20000
--
--
08-06 20:00:32.916  1267  1386 V WindowManager:   pfl=20000
08-06 20:00:32.916  1267  1386 V WindowManager:   vsysui=2000 naviIconColor=0}, display=0
08-06 20:00:32.917 29292 29292 I ViewRootImpl@bc94799[LinkActivity]: Relayout returned: old=(0,0,1080,2220) new=(0,0,1080,2220) req=(1080,2220)0 dur=10 res=0x7 s={true 491343003648} ch=true
08-06 20:00:32.918  1267  1490 I WindowManager: Focus moving from null to Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity} displayId=0
08-06 20:00:32.918  1267  1490 I WindowManager: Gaining focus: Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity}
08-06 20:00:32.921  1267  1386 D PackageManager: setEnabledSetting : userId = 0 packageName = com.titannative cmp = androidx.work.impl.background.systemalarm.RescheduleReceiver newState = 2 callingPackage = 10739/com.titannative
--
08-06 20:00:32.921  1267  1386 D PackageManager: setEnabledSetting : userId = 0 packageName = com.titannative cmp = androidx.work.impl.background.systemalarm.RescheduleReceiver newState = 2 callingPackage = 10739/com.titannative
08-06 20:00:32.927  1267  1386 D WindowManager: finishDrawingWindow: Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity} mDrawState=DRAW_PENDING
08-06 20:00:32.927  1267  4769 D JobServiceContext: sendStopMessageLocked stop reason: null
--
08-06 20:00:32.927  1267  4769 D JobServiceContext: sendStopMessageLocked stop reason: null
08-06 20:00:32.930  1267  1491 D PkgPredictorService: pkg:com.titannative activity:com.plaid.link.LinkActivity thisTime:-1
08-06 20:00:32.930  1267  1491 V WindowManager: performShow on Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity}: mDrawState=READY_TO_SHOW readyForDisplay=false starting=false during animation: policyVis=true parentHidden=false tok.hiddenRequested=false tok.hidden=true animating=true tok animating=true Callers=com.android.server.wm.WindowState.performShowLocked:4819 com.android.server.wm.WindowStateAnimator.commitFinishDrawingLocked:415 com.android.server.wm.DisplayContent.lambda$new$8$DisplayContent:1026 com.android.server.wm.-$$Lambda$DisplayContent$qxt4izS31fb0LF2uo_OF9DMa7gc.accept:4
08-06 20:00:32.932  1267  1386 D PackageManager: setEnabledSetting : userId = 0 packageName = com.titannative cmp = androidx.work.impl.background.systemalarm.RescheduleReceiver newState = 2 callingPackage = 10739/com.titannative
--
08-06 20:00:32.932  1267  1386 D PackageManager: setEnabledSetting : userId = 0 packageName = com.titannative cmp = androidx.work.impl.background.systemalarm.RescheduleReceiver newState = 2 callingPackage = 10739/com.titannative
08-06 20:00:32.933  1267  1491 V WindowManager: performShow on Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity}: mDrawState=READY_TO_SHOW readyForDisplay=false starting=false during animation: policyVis=true parentHidden=false tok.hiddenRequested=false tok.hidden=true animating=true tok animating=true Callers=com.android.server.wm.WindowState.performShowLocked:4819 com.android.server.wm.DisplayContent.lambda$new$0$DisplayContent:705 com.android.server.wm.-$$Lambda$DisplayContent$0yxrqH9eGY2qTjH1u_BvaVrXCSA.accept:4 com.android.server.wm.WindowContainer$ForAllWindowsConsumerWrapper.apply:1188
08-06 20:00:32.933  1267  4576 D SensorService: removeConnection, index : 3, size : 5
08-06 20:00:32.933  1267  4576 D SensorService: # removeConnection, index : 3, size : 4
08-06 20:00:32.933  1267  4576 D SensorService: Calling activate off 28
--
--
08-06 20:00:32.933  1267  4576 D SensorService: Calling activate off 28
08-06 20:00:32.933   837   898 I sensors-hal: batch:276, android.sensor.accelerometer/28, period=20000000, max_latency=66667000 request completed
08-06 20:00:32.934 29292 29292 D SensorManager: unregisterListener ::
08-06 20:00:32.935  1267  1491 V WindowManager: performShow on Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity}: mDrawState=READY_TO_SHOW readyForDisplay=false starting=false during animation: policyVis=true parentHidden=false tok.hiddenRequested=false tok.hidden=true animating=true tok animating=true Callers=com.android.server.wm.WindowState.performShowLocked:4819 com.android.server.wm.WindowStateAnimator.commitFinishDrawingLocked:415 com.android.server.wm.DisplayContent.lambda$new$8$DisplayContent:1026 com.android.server.wm.-$$Lambda$DisplayContent$qxt4izS31fb0LF2uo_OF9DMa7gc.accept:4
08-06 20:00:32.936  1267  4576 D SamsungAlarmManager: Cancel Alarm calling from uid:10031 pid :25393 / op:PendingIntent{c5e8b63: PendingIntentRecord{bb4619c com.google.android.gms broadcastIntent}}
08-06 20:00:32.938  1267  1386 V WindowManager: Prepare app transition: transit=TRANSIT_ACTIVITY_CLOSE mNextAppTransition=TRANSIT_ACTIVITY_OPEN alwaysKeepCurrent=false displayId=0 Callers=com.android.server.wm.DisplayContent.prepareAppTransition:6356 com.android.server.wm.DisplayContent.prepareAppTransition:6350 com.android.server.wm.ActivityStack.resumeTopActivityInnerLocked:3535 com.android.server.wm.ActivityStack.resumeTopActivityUncheckedLocked:3154 com.android.server.wm.RootActivityContainer.resumeFocusedStacksTopActivities:1246
08-06 20:00:32.939  1267  1386 D TspStateManagerInternal: updateDefaultValue customSetting=null
--
--
08-06 20:00:32.941  1267  2036 D GameManagerService: onLooperPrepared(), msg: { when=-1ms what=10010 obj=com.titannative target=com.samsung.android.game.GameManagerService$FgCheckThread$1 }
08-06 20:00:32.941  1267  2036 D GameManagerService: handleMessage(), received MSG_APP_CREATE of pkgName: com.titannative
08-06 20:00:32.941  1267  1386 D InputDispatcher: Focused application set to (0): c4419c7
08-06 20:00:32.942  1267  1386 V WindowManager: Changing focus from Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity} to null displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:204 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6458 com.android.server.wm.ActivityDisplay.setFocusedApp:1686 com.android.server.wm.ActivityTaskManagerService.setResumedActivityUncheckLocked:6980 com.android.server.wm.ActivityStack.onActivityStateChanged:664 com.android.server.wm.TaskRecord.onActivityStateChanged:2425
08-06 20:00:32.942  1267  1488 D WindowManager: setSystemUiVisibility: displayId=0, vis=0xa008, fullVis=0x2000, dockVis=0x0, win=Window{33ca449 u0 com.titannative/com.titannative.MainActivity}
08-06 20:00:32.942  1267  1386 I SemPersonaManager: SecureFolder personaId = 11
08-06 20:00:32.942  1267  1386 D MARsPolicyManager: onPackageResumedFG pkgName = com.titannative, userId = 0
--
--
08-06 20:00:32.950  1267  2115 D SamsungAlarmManager: Cancel Alarm calling from uid:10031 pid :25393 / op:PendingIntent{122d019: PendingIntentRecord{bb4619c com.google.android.gms broadcastIntent}}
08-06 20:00:32.954 29292 29292 I ViewRootImpl@bc94799[LinkActivity]: MSG_RESIZED: frame=(0,0,1080,2220) ci=(0,63,0,0) vi=(0,63,0,0) or=1
08-06 20:00:32.954 29292 29292 W ActivityThread: handleWindowVisibility: no activity for token android.os.BinderProxy@bdc070e
08-06 20:00:32.957  1267  1490 I WindowManager: Focus moving from Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity} to null displayId=0
08-06 20:00:32.957  1267  1490 I WindowManager: Losing focus: Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity}
08-06 20:00:32.959 29292 29292 D PhoneWindow: forceLight changed to true [] from com.android.internal.policy.PhoneWindow.updateForceLightNavigationBar:4274 com.android.internal.policy.DecorView.updateColorViews:1547 com.android.internal.policy.PhoneWindow.dispatchWindowAttributesChanged:3252 android.view.Window.setFlags:1153 com.android.internal.policy.PhoneWindow.generateLayout:2474
08-06 20:00:32.960 29292 29292 I MultiWindowDecorSupport: [INFO] isPopOver = false
08-06 20:00:32.960 29292 29292 I MultiWindowDecorSupport: updateCaptionType >> DecorView@7b56f7d[], isFloating: false, isApplication: true, hasWindowDecorCaption: false, hasWindowControllerCallback: true
--
--
08-06 20:00:32.960 29292 29292 I MultiWindowDecorSupport: updateCaptionType >> DecorView@7b56f7d[], isFloating: false, isApplication: true, hasWindowDecorCaption: false, hasWindowControllerCallback: true
08-06 20:00:32.960 29292 29292 D MultiWindowDecorSupport: setCaptionType = 0, DecorView = DecorView@7b56f7d[]
08-06 20:00:32.962  1267  1490 D GameManagerService: noteResumeComponent(), received resumed-component: com.titannative
08-06 20:00:32.962  1267  2036 D GameManagerService: onLooperPrepared(), msg: { when=0 what=10000 obj=ComponentInfo{com.titannative/com.plaid.internal.link.LinkWebViewActivity} target=com.samsung.android.game.GameManagerService$FgCheckThread$1 }
08-06 20:00:32.962  1267  2036 D GameManagerService: sendPauseConditionally() - mPrevNotiPackage: com.sec.android.app.clockpackage
08-06 20:00:32.962  1267  2036 D GameManagerService: notifyResumePause(). of pkg: com.titannative, type: 4, isMinimized: false, isTunableApp: false
08-06 20:00:32.962  1267  2036 D GameManagerService:   notifyResumePause(). do nothing. mKillNotiCount: 1
--
--
08-06 20:00:32.984  1267  2115 E WindowManager: App trying to use insecure INPUT_FEATURE_NO_INPUT_CHANNEL flag. Ignoring
08-06 20:00:32.984  1267  2115 D InputTransport: Input channel constructed: '1a8a58c', fd=905
08-06 20:00:32.984  1267  2115 D InputTransport: Input channel constructed: '1a8a58c', fd=907
08-06 20:00:32.984   914   999 I SurfaceFlinger: id=45267 createSurf (0x0),-1 flag=80004, 1a8a58c com.titannative/com.plaid.internal.link.LinkWebViewActivity#0
08-06 20:00:32.984  6157  6157 W SmsApplication: Package com.vcast.mediamanager lacks required manifest declarations to be a default sms app:  mPackageName: com.vcast.mediamanager mSmsReceiverClass: com.verizon.contenttransfer.sms.DefaultSMSReceiver mMmsReceiverClass: null mRespondViaMessageClass: com.verizon.contenttransfer.sms.HeadlessSmsSendService mSendToClass: null mSmsAppChangedClass: null mProviderChangedReceiverClass: null mSimFullReceiverClass: null mUid: 10094
08-06 20:00:32.985  6157  6157 W SmsApplication: Package com.vzw.hss.myverizon lacks required manifest declarations to be a default sms app:  mPackageName: com.vzw.hss.myverizon mSmsReceiverClass: com.verizon.contenttransfer.sms.DefaultSMSReceiver mMmsReceiverClass: null mRespondViaMessageClass: com.verizon.contenttransfer.sms.HeadlessSmsSendService mSendToClass: null mSmsAppChangedClass: null mProviderChangedReceiverClass: null mSimFullReceiverClass: null mUid: 10108
08-06 20:00:32.985  6157  6157 W SmsApplication: Package com.google.android.talk lacks required manifest declarations to be a default sms app:  mPackageName: com.google.android.talk mSmsReceiverClass: com.google.android.apps.hangouts.sms.SmsDeliverReceiver mMmsReceiverClass: com.google.android.apps.hangouts.sms.MmsWapPushDeliverReceiver mRespondViaMessageClass: null mSendToClass: com.google.android.apps.hangouts.phone.BabelHomeActivity mSmsAppChangedClass: null mProviderChangedReceiverClass: null mSimFullReceiverClass: null mUid: 10324
--
--
08-06 20:00:32.985  6157  6157 W SmsApplication: Package com.google.android.talk lacks required manifest declarations to be a default sms app:  mPackageName: com.google.android.talk mSmsReceiverClass: com.google.android.apps.hangouts.sms.SmsDeliverReceiver mMmsReceiverClass: com.google.android.apps.hangouts.sms.MmsWapPushDeliverReceiver mRespondViaMessageClass: null mSendToClass: com.google.android.apps.hangouts.phone.BabelHomeActivity mSmsAppChangedClass: null mProviderChangedReceiverClass: null mSimFullReceiverClass: null mUid: 10324
08-06 20:00:32.985  1267  2115 D InputTransport: Input channel destroyed: '1a8a58c', fd=907
08-06 20:00:32.986 29292 29292 I ViewRootImpl@95cb559[LinkWebViewActivity]: setView = com.android.internal.policy.DecorView@7b56f7d TM=true MM=false
08-06 20:00:32.990  1267  4181 V WindowManager: Relayout Window{1a8a58c u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity}: viewVisibility=0 req=1080x2220
08-06 20:00:32.993   914  1099 I SurfaceFlinger: id=45268 createSurf (1080x2220),1 flag=404, com.titannative/com.plaid.internal.link.LinkWebViewActivity$_29292#0
08-06 20:00:32.994  1267  4181 D WindowManager: makeSurface duration=1 name=com.titannative/com.plaid.internal.link.LinkWebViewActivity$_29292
08-06 20:00:32.994  1267  4181 V WindowManager: Changing focus from null to Window{1a8a58c u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity} displayId=0 Callers=com.android.server.wm.RootWindowContainer.updateFocusedWindowLocked:204 com.android.server.wm.WindowManagerService.updateFocusedWindowLocked:6458 com.android.server.wm.WindowManagerService.relayoutWindow:2618 com.android.server.wm.Session.relayout:202 android.view.IWindowSession$Stub.onTransact:716 com.android.server.wm.Session.onTransact:142
08-06 20:00:32.994  1267  4181 D StatusBarManagerService: notifyRequestedSystemKey recent=false home=false
--
08-06 20:00:32.994  1267  4181 D StatusBarManagerService: notifyRequestedSystemKey recent=false home=false
08-06 20:00:32.994  1267  1488 D WindowManager: setSystemUiVisibility: displayId=0, vis=0x8008, fullVis=0x2000, dockVis=0x0, win=Window{1a8a58c u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity}
08-06 20:00:32.995  1267  1490 I WindowManager: Focus moving from null to Window{1a8a58c u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity} displayId=0
08-06 20:00:32.995  1267  1490 I WindowManager: Gaining focus: Window{1a8a58c u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity}
08-06 20:00:32.995  1267  4181 V WindowManager: Relayout 29292: mAttrs={(0,0)(fillxfill) sim={adjust=pan forwardNavigation} ty=BASE_APPLICATION wanim=0x10302fe
08-06 20:00:32.995  1267  4181 V WindowManager:   fl=1810100
08-06 20:00:32.995  1267  4181 V WindowManager:   pfl=20000 naviIconColor=0}, display=0
--
--
08-06 20:00:32.995  1267  4181 V WindowManager:   pfl=20000 naviIconColor=0}, display=0
08-06 20:00:32.996  1267  4181 D SamsungAlarmManager: Cancel Alarm calling from uid:10031 pid :25393 / op:PendingIntent{d8754d5: PendingIntentRecord{bb4619c com.google.android.gms broadcastIntent}}
08-06 20:00:32.996 29292 29292 I ViewRootImpl@95cb559[LinkWebViewActivity]: Relayout returned: old=(0,0,1080,2220) new=(0,0,1080,2220) req=(1080,2220)0 dur=6 res=0x7 s={true 488680058880} ch=true
08-06 20:00:33.002  1267  4181 D WindowManager: finishDrawingWindow: Window{1a8a58c u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity} mDrawState=DRAW_PENDING
08-06 20:00:33.004 29292 29292 I ViewRootImpl@bc94799[LinkActivity]: MSG_WINDOW_FOCUS_CHANGED 0 1
08-06 20:00:33.004 29292 29292 D InputMethodManager: prepareNavigationBarInfo() DecorView@c8825d6[LinkActivity]
08-06 20:00:33.004 29292 29292 D InputMethodManager: getNavigationBarColor() -14735316
--
--
08-06 20:00:33.004 29292 29292 I ViewRootImpl@bc94799[LinkActivity]: MSG_WINDOW_FOCUS_CHANGED 0 1
08-06 20:00:33.004 29292 29292 D InputMethodManager: prepareNavigationBarInfo() DecorView@c8825d6[LinkActivity]
08-06 20:00:33.004 29292 29292 D InputMethodManager: getNavigationBarColor() -14735316
08-06 20:00:33.005  1267  1491 V WindowManager: Setting visibility of Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity}: false, caller=com.android.server.wm.WindowContainer.sendAppVisibilityToClients:644 com.android.server.wm.AppWindowToken.setClientHidden:632 com.android.server.wm.AppWindowToken.onAnimationFinished:3913 com.android.server.wm.AppWindowToken.commitVisibility:914 com.android.server.wm.AppTransitionController.handleClosingApps:448
08-06 20:00:33.005  2121  2121 D LightBarController: onNavigationVisibilityChanged : vis = 8008, nbModeChanged = false navigationBarMode = 4, navbarColorManagedByIme = false
08-06 20:00:33.005  6637 30643 W NetworkScheduler: Error inserting flex_time=2448000 job_id=-1 period=4898000 source=16 requires_charging=0 preferred_network_type=1 target_class=com.google.android.gms.measurement.PackageMeasurementTaskService user_id=0 target_package=com.google.android.gms tag=Measurement.PackageMeasurementTaskService.UPLOAD_TASK_TAG task_type=0 required_idleness_state=0 service_kind=0 source_version=212621000 persistence_level=1 preferred_charging_state=1 required_network_type=0 runtime=1628294433003 retry_strategy={"maximum_backoff_seconds":{"3600":0},"initial_backoff_seconds":{"30":0},"retry_policy":{"0":0}} last_runtime=0, error message: UNIQUE constraint failed: pending_ops.tag, pending_ops.target_class, pending_ops.target_package, pending_ops.user_id (code 2067 SQLITE_CONSTRAINT_UNIQUE[2067]) [CONTEXT service_id=218 ]
08-06 20:00:33.005  1267  1491 W WindowManager: requestTraversal called while in layout. Callers=com.android.server.wm.WindowManagerService.requestTraversal:6445 com.android.server.wm.WindowState.onExitAnimationDone:5148 com.android.server.wm.-$$Lambda$01bPtngJg5AqEoOWfW3rWfV7MH4.accept:2
--
--
08-06 20:00:33.009  1267  1491 I WindowManager: Reparenting to leash, surface=Surface(name=Dim Layer for - Task=29895)/@0x347742
08-06 20:00:33.009   914  2739 I SurfaceFlinger: id=45272 createSurf (0x0),-1 flag=80004, Surface(name=Dim Layer for - Task=29895)/@0x347742 - animation-leash#0
08-06 20:00:33.009  1267  1491 D WindowManager: makeSurface duration=1 leash=Surface(name=Surface(name=Dim Layer for - Task=29895)/@0x347742 - animation-leash)/@0x1b91353
08-06 20:00:33.010  1267  1491 I WindowManager: Reparenting to leash, surface=Surface(name=AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}})/@0x9bdac9a
08-06 20:00:33.010   914  2739 I SurfaceFlinger: id=45273 createSurf (0x0),-1 flag=80004, Surface(name=AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}})/@0x9bdac9a - animation-leash#0
08-06 20:00:33.011  1267  1491 D WindowManager: makeSurface duration=0 leash=Surface(name=Surface(name=AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}})/@0x9bdac9a - animation-leash)/@0x97599cb
08-06 20:00:33.011   914  2739 I SurfaceFlinger: id=45274 createSurf (0x0),-1 flag=80004, Surface(name=AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}})/@0x9bdac9a - animation-bounds#0
08-06 20:00:33.012  1267  1491 D PkgPredictorService: pkg:com.titannative activity:com.plaid.internal.link.LinkWebViewActivity thisTime:-1
08-06 20:00:33.014  1267  1488 D WindowManager: setSystemUiVisibility: displayId=0, vis=0x8008, fullVis=0x0, dockVis=0x0, win=Window{1a8a58c u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity}
08-06 20:00:33.015  1267  1386 D WindowManager: adjustSystemUiVisibilityLw: displayId=0, vis=0x8008, resetVis=0x0, clearVis=0x0
08-06 20:00:33.015  1267  1386 D InputManager-JNI: setSystemUiVisibility dexmode: false changed: true displayid: 0 visibility: 0x8008 DD: 0x8008 OD: 0x0
08-06 20:00:33.015 10590 10590 I TrayStateController: onSystemUiVisibilityChange : 32776 0
--
--
08-06 20:00:33.021  6960  6960 I HONEYBOARD: a [UpdatePolicy] [post] a: 0 cl: 65537 kit: [0/0] vt: 0 ir: 1
08-06 20:00:33.021  6960  6960 I HONEYBOARD: HoneyBoardService [PF_OP][onStartInput]  2278698 ns
08-06 20:00:33.032  1267  2115 D InputDispatcher: Focus entered window (29292): 1a8a58c in display 0 0
08-06 20:00:33.032  1267  4576 V WindowManager: Relayout Window{1e4c9eb u0 com.titannative/com.plaid.link.LinkActivity}: viewVisibility=8 req=1080x2220
08-06 20:00:33.033  1267  1386 V WindowManager: Relayout Window{2f093cb u0 StatusBar}: viewVisibility=0 req=1080x63
08-06 20:00:33.034  1267  4576 V WindowManager: Relayout 29292: mAttrs={(0,0)(fillxfill) sim={adjust=pan forwardNavigation} ty=BASE_APPLICATION wanim=0x1030305
08-06 20:00:33.034  1267  4576 V WindowManager:   fl=81810100
--
--
08-06 20:00:33.038 29292 29292 I ViewRootImpl@bc94799[LinkActivity]: stopped(true) old=false
08-06 20:00:33.041 29292 29292 I ViewRootImpl@bc94799[LinkActivity]: dispatchDetachedFromWindow
08-06 20:00:33.041  1267  4576 D InputTransport: Input channel destroyed: '1e4c9eb', fd=275
08-06 20:00:33.042  1267  4576 I WindowManager: Destroying surface Surface(name=com.titannative/com.plaid.link.LinkActivity$_29292)/@0x39c5454 called by com.android.server.wm.WindowStateAnimator.destroySurface:1834 com.android.server.wm.WindowStateAnimator.destroySurfaceLocked:733 com.android.server.wm.WindowState.removeImmediately:2402 com.android.server.wm.WindowState.removeIfPossible:2546 com.android.server.wm.WindowState.removeIfPossible:2425 com.android.server.wm.WindowManagerService.removeWindow:2086 com.android.server.wm.Session.remove:184 android.view.IWindowSession$Stub.onTransact:666
08-06 20:00:33.042  2121  2121 E [IndicatorGarden]Presenter: onGardenApplyWindowInsets() New DisplayCutout is NULL!!
--
08-06 20:00:33.042  2121  2121 E [IndicatorGarden]Presenter: onGardenApplyWindowInsets() New DisplayCutout is NULL!!
08-06 20:00:33.042   914  2854 I Layer   : id=45266 removeFromCurrentState com.titannative/com.plaid.link.LinkActivity$_29292#0 (132)
08-06 20:00:33.044   914  2854 I Layer   : id=45264 removeFromCurrentState 1e4c9eb com.titannative/com.plaid.link.LinkActivity#0 (132)
08-06 20:00:33.044   914  2854 I SurfaceFlinger: id=45266 Removed com.titannative/com.plaid.link.LinkActivity$_29292#0 (132)
08-06 20:00:33.044   914  2854 I SurfaceFlinger: id=45264 Removed 1e4c9eb com.titannative/com.plaid.link.LinkActivity#0 (132)
08-06 20:00:33.044 29292 29292 D InputTransport: Input channel destroyed: '1e4c9eb', fd=143
--
08-06 20:00:33.044 29292 29292 D InputTransport: Input channel destroyed: '1e4c9eb', fd=143
08-06 20:00:33.047   914   914 I Layer   : id=45264[1] Destroyed 1e4c9eb com.titannative/com.plaid.link.LinkActivity#0
08-06 20:00:33.047   914   914 I Layer   : id=45266[1] Destroyed com.titannative/com.plaid.link.LinkActivity$_29292#0
08-06 20:00:33.048   914   999 I Layer   : id=45262 removeFromCurrentState AppWindowToken{2ac4cd7 token=Token{e5a5456 ActivityRecord{6212418 u0 com.titannative/com.plaid.link.LinkActivity t29895}}}#0 (130)
08-06 20:00:33.048   914  2854 I SurfaceFlinger: id=45262 Removed AppWindowToken{2ac4cd7 token=Token{e5a5456 ActivityRecord{6212418 u0 com.titannative/com.plaid.link.LinkActivity t29895}}}#0 (130)
08-06 20:00:33.054   914   914 I SurfaceFlinger: Display 19261213734341249 HWC layers:
08-06 20:00:33.054   914   914 I SurfaceFlinger:     type    |    handle    | flag |  format   |   source crop (l,t,r,b)    |         frame       | name
08-06 20:00:33.054   914   914 I SurfaceFlinger: ------------+--------------+------+-----------+----------------------------+---------------------+------
--
--
08-06 20:00:33.054   914   914 I SurfaceFlinger: ------------+--------------+------+-----------+----------------------------+---------------------+------
08-06 20:00:33.054   914   914 I SurfaceFlinger:      DEVICE | 0x75b4e4a7e0 | 0002 | RGBA_8888 |   0.0   18.0 1080.0 2220.0 |    0    0 1080 2202 | com.titannative/com.titannative.MainActivity$_29292#0
08-06 20:00:33.054   914   914 I SurfaceFlinger:  SOLID_COLOR |              | 0004 |  Unknown  |   0.0    0.0    0.0    0.0 |    0    0 1080 2220 | Dim Layer for - Task=29895#0
08-06 20:00:33.054   914   914 I SurfaceFlinger:      DEVICE | 0x75b4e4aee0 | 0002 | RGBA_8888 |   0.0    0.0 1080.0 1912.0 |    0  308 1080 2220 | com.titannative/com.plaid.internal.link.LinkWebViewActivity$_29292#0
08-06 20:00:33.054   914   914 I SurfaceFlinger:      DEVICE | 0x75b4e4b260 | 0000 | RGBA_8888 |   0.0    0.0 1080.0   63.0 |    0    0 1080   63 | StatusBar$_2121#0
08-06 20:00:33.054   914   914 I SurfaceFlinger:      DEVICE | 0x75a62e92d0 | 0000 | RGBA_8888 |   0.0    0.0   67.0  433.0 |    0 1420   67 1853 | com.samsung.android.app.cocktailbars[...]rservice.CocktailBarService$_10590#0
08-06 20:00:33.054   914   914 I SurfaceFlinger:      DEVICE | 0x75b4e4ae00 | 00
--
--
08-06 20:00:33.054   914   914 I SurfaceFlinger: 00 | RGBA_8888 |   0.0    0.0 1080.0   84.0 |    0 2136 1080 2220 | GestureInterceptOverlay$_2121#0
08-06 20:00:33.054   914   914 I SurfaceFlinger:      DEVICE | 0x75b8f20da0 | 0000 | RGBA_8888 |   0.0    0.0 1080.0  126.0 |    0 2094 1080 2220 | NavigationBar0$_2121#0
08-06 20:00:33.054   914   914 I SurfaceFlinger:
08-06 20:00:33.064   914   914 I Layer   : id=45262[1] Destroyed AppWindowToken{2ac4cd7 token=Token{e5a5456 ActivityRecord{6212418 u0 com.titannative/com.plaid.link.LinkActivity t29895}}}#0
08-06 20:00:33.213  1267  1491 I WindowManager: Reparenting to original parent: Surface(name=Task=29895)/@0x9ea976b, destroyLeash=true, surface=Surface(name=Dim Layer for - Task=29895)/@0x347742
08-06 20:00:33.231   914  2739 I Layer   : id=45272 removeFromCurrentState Surface(name=Dim Layer for - Task=29895)/@0x347742 - animation-leash#0 (129)
08-06 20:00:33.231   914  2854 I SurfaceFlinger: id=45272 Removed Surface(name=Dim Layer for - Task=29895)/@0x347742 - animation-leash#0 (129)
--
--
08-06 20:00:33.371  3783  3827 E chromium: [ERROR:tile_manager.cc(821)] WARNING: tile memory limits exceeded, some content may not draw
08-06 20:00:33.397  1267  1491 I WindowManager: Reparenting to original parent: Surface(name=Task=29895)/@0x9ea976b, destroyLeash=true, surface=Surface(name=AppWindowToken{7c9e147 token=Token{f38b486 ActivityRecord{463e161 u0 com.titannative/.MainActivity t29895}}})/@0x826e533
08-06 20:00:33.398  1267  1491 V WindowManager: Setting visibility of Window{33ca449 u0 com.titannative/com.titannative.MainActivity}: false, caller=com.android.server.wm.WindowContainer.sendAppVisibilityToClients:644 com.android.server.wm.AppWindowToken.setClientHidden:632 com.android.server.wm.AppWindowToken.onAnimationFinished:3913 com.android.server.wm.-$$Lambda$yVRF8YoeNdTa8GR1wDStVsHu8xM.run:2 com.android.server.wm.SurfaceAnimator.lambda$getFinishedCallback$0$SurfaceAnimator:100
08-06 20:00:33.398  1267  1491 I WindowManager: Reparenting to original parent: Surface(name=Task=29895)/@0x9ea976b, destroyLeash=true, surface=Surface(name=AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}})/@0x9bdac9a
08-06 20:00:33.398  1267  1491 D CustomFrequencyManagerService: acquireDVFSLockLocked : type : DVFS_MIN_LIMIT  frequency : 1689600  uid : 1000  pid : 1267  pkgName : AMS_RESUME_TAIL@CPU_MIN@30
08-06 20:00:33.398  1267  1491 D ActivityManagerPerformance: AMP_acquire() TAIL
08-06 20:00:33.398  1267  1491 D CustomFrequencyManagerService: FrequencyrequestList.getNextMaxLPMBiasRequest, index: 8
--
--
08-06 20:00:33.402   914   914 I SurfaceFlinger: Display 19261213734341249 HWC layers:
08-06 20:00:33.402   914   914 I SurfaceFlinger:     type    |    handle    | flag |  format   |   source crop (l,t,r,b)    |         frame       | name
08-06 20:00:33.402   914   914 I SurfaceFlinger: ------------+--------------+------+-----------+----------------------------+---------------------+------
08-06 20:00:33.402   914   914 I SurfaceFlinger:      DEVICE | 0x75b4e4a070 | 0002 | RGBA_8888 |   0.0    0.0 1080.0 2220.0 |    0    0 1080 2220 | com.titannative/com.plaid.internal.link.LinkWebViewActivity$_29292#0
08-06 20:00:33.402   914   914 I SurfaceFlinger:      DEVICE | 0x75b4e4ac40 | 0000 | RGBA_8888 |   0.0    0.0 1080.0   63.0 |    0    0 1080   63 | StatusBar$_2121#0
08-06 20:00:33.402   914   914 I SurfaceFlinger:      DEVICE | 0x75a62e92d0 | 0000 | RGBA_8888 |   0.0    0.0   67.0  433.0 |    0 1420   67 1853 | com.samsung.android.app.cocktailbars[...]rservice.CocktailBarService$_10590#0
08-06 20:00:33.402   914   914 I SurfaceFlinger:      DEVICE | 0x75b4e4ae00 | 0000 | RGBA_8888 |   0.0    0.0 1080.0   84.0 |    0 2136 1080 2220 | GestureInterceptOverlay$_2121#0
--
--
08-06 20:00:33.402   914   914 I SurfaceFlinger:
08-06 20:00:33.404   914  2739 I Layer   : id=45269 removeFromCurrentState Surface(name=AppWindowToken{7c9e147 token=Token{f38b486 ActivityRecord{463e161 u0 com.titannative/.MainActivity t29895}}})/@0x826e533 - animation-leash#0 (128)
08-06 20:00:33.404   914  2739 I Layer   : id=45270 removeFromCurrentState Surface(name=AppWindowToken{7c9e147 token=Token{f38b486 ActivityRecord{463e161 u0 com.titannative/.MainActivity t29895}}})/@0x826e533 - animation-bounds#0 (128)
08-06 20:00:33.404   914  2739 I Layer   : id=45265 removeFromCurrentState AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}}#0 (128)
08-06 20:00:33.404   914  2739 I Layer   : id=45268 removeFromCurrentState com.titannative/com.plaid.internal.link.LinkWebViewActivity$_29292#0 (128)
08-06 20:00:33.404   914  2739 I Layer   : id=45267 removeFromCurrentState 1a8a58c com.titannative/com.plaid.internal.link.LinkWebViewActivity#0 (128)
08-06 20:00:33.404   914  2739 I Layer   : id=45273 removeFromCurrentState Surface(name=AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}})/@0x9bdac9a - animation-leash#0 (128)
08-06 20:00:33.404   914  2739 I Layer   : id=45274 removeFromCurrentState Surface(name=AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}})/@0x9bdac9a - animation-bounds#0 (128)
08-06 20:00:33.404   914  2739 I Layer   : id=45271 removeFromCurrentState Dim Layer for - Task=29895#0 (128)
--
08-06 20:00:33.404   914  2739 I Layer   : id=45271 removeFromCurrentState Dim Layer for - Task=29895#0 (128)
08-06 20:00:33.405   914  2854 I SurfaceFlinger: id=45274 Removed Surface(name=AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}})/@0x9bdac9a - animation-bounds#0 (128)
08-06 20:00:33.405   914  2739 I SurfaceFlinger: id=45269 Removed Surface(name=AppWindowToken{7c9e147 token=Token{f38b486 ActivityRecord{463e161 u0 com.titannative/.MainActivity t29895}}})/@0x826e533 - animation-leash#0 (128)
08-06 20:00:33.405   914  2739 I SurfaceFlinger: id=45270 Removed Surface(name=AppWindowToken{7c9e147 token=Token{f38b486 ActivityRecord{463e161 u0 com.titannative/.MainActivity t29895}}})/@0x826e533 - animation-bounds#0 (128)
--
08-06 20:00:33.405   914  2739 I SurfaceFlinger: id=45269 Removed Surface(name=AppWindowToken{7c9e147 token=Token{f38b486 ActivityRecord{463e161 u0 com.titannative/.MainActivity t29895}}})/@0x826e533 - animation-leash#0 (128)
08-06 20:00:33.405   914  2739 I SurfaceFlinger: id=45270 Removed Surface(name=AppWindowToken{7c9e147 token=Token{f38b486 ActivityRecord{463e161 u0 com.titannative/.MainActivity t29895}}})/@0x826e533 - animation-bounds#0 (128)
08-06 20:00:33.405   914  2739 I SurfaceFlinger: id=45273 Removed Surface(name=AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}})/@0x9bdac9a - animation-leash#0 (128)
08-06 20:00:33.405   914  2854 I SurfaceFlinger: id=45271 Removed Dim Layer for - Task=29895#0 (128)
08-06 20:00:33.414   914   914 I Layer   : id=45270[1] Destroyed Surface(name=AppWindowToken{7c9e147 token=Token{f38b486 ActivityRecord{463e161 u0 com.titannative/.MainActivity t29895}}})/@0x826e533 - animation-bounds#0
08-06 20:00:33.414   914   914 I Layer   : id=45269[1] Destroyed Surface(name=AppWindowToken{7c9e147 token=Token{f38b486 ActivityRecord{463e161 u0 com.titannative/.MainActivity t29895}}})/@0x826e533 - animation-leash#0
--
--
08-06 20:00:33.405   914  2854 I SurfaceFlinger: id=45271 Removed Dim Layer for - Task=29895#0 (128)
08-06 20:00:33.414   914   914 I Layer   : id=45270[1] Destroyed Surface(name=AppWindowToken{7c9e147 token=Token{f38b486 ActivityRecord{463e161 u0 com.titannative/.MainActivity t29895}}})/@0x826e533 - animation-bounds#0
08-06 20:00:33.414   914   914 I Layer   : id=45269[1] Destroyed Surface(name=AppWindowToken{7c9e147 token=Token{f38b486 ActivityRecord{463e161 u0 com.titannative/.MainActivity t29895}}})/@0x826e533 - animation-leash#0
08-06 20:00:33.415   914   914 I Layer   : id=45274[1] Destroyed Surface(name=AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}})/@0x9bdac9a - animation-bounds#0
08-06 20:00:33.415   914   914 I Layer   : id=45273[1] Destroyed Surface(name=AppWindowToken{c4419c7 token=Token{9ce7f06 ActivityRecord{8e045e1 u0 com.titannative/com.plaid.internal.link.LinkWebViewActivity t29895}}})/@0x9bdac9a - animation-leash#0
08-06 20:00:33.415   914   914 I Layer   : id=45271[1] Destroyed Dim Layer for - Task=29895#0
08-06 20:00:33.418  1267  1386 V WindowManager: Relayout Window{33ca449 u0 com.titannative/com.titannative.MainActivity}: viewVisibility=8 req=1080x2220
08-06 20:00:33.420  1267  1386 V WindowManager: Relayout 29292: mAttrs={(0,0)(fillxfill) sim={adjust=resize} ty=BASE_APPLICATION wanim=0x1030305
--
--
08-06 20:00:33.444  3783  3827 E chromium: [ERROR:tile_manager.cc(821)] WARNING: tile memory limits exceeded, some content may not draw
08-06 20:00:33.448   914   914 I Layer   : id=45250[1] Destroyed com.titannative/com.titannative.MainActivity$_29292#0
08-06 20:00:33.700  1267  1267 D CustomFrequencyManagerService: releaseDVFSLockLocked : Getting Lock type frm List : DVFS_MIN_LIMIT  frequency : 1689600  uid : 1000  pid : 1267  tag : AMS_RESUME_TAIL@CPU_MIN@30
08-06 20:00:33.750 29292 26180 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=8756ccc1-dcd4-419f-a2d6-447ed736a2e1, tags={ com.plaid.internal.core.analytics.batch.AnalyticsBatchUploadWorker } ]
08-06 20:00:33.752  1267  2115 D PackageManager: setEnabledSetting : userId = 0 packageName = com.titannative cmp = androidx.work.impl.background.systemalarm.RescheduleReceiver newState = 2 callingPackage = 10739/com.titannative
08-06 20:00:33.791  3783  3827 E chromium: [ERROR:tile_manager.cc(821)] WARNING: tile memory limits exceeded, some content may not draw
08-06 20:00:33.796  3783  3827 I chatty  : uid=99516(u0_i9516) Compositor identical 2 lines
```

**Comment by Crizzooo** [comment]
Hi @amytang0 !

I upgraded to 7.0.7. Good news is that it doesnt infinitely spin, bad news is that I received this screenshot below.

Hopefully with this screenshot its easier to detect what is wrong and how to fix?

![Screenshot_20210806-204809_Titan](https://user-images.githubusercontent.com/18145174/128582662-b18f5943-e4a5-4930-a6c3-548c677123a8.jpg)

Also noting that I had to add `classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.10'`  to my `android/build.gradle` to be able to build from RN to android

**Comment by amytang0** [comment]
That is very odd, does this still only work with the debug variant/ fail with release?

Do you also mind running `./gradlew :<yourAppHere>:dependencies | grep sdk-core` to see what Android dependency is actually being pulled in? 

We can preemptively rule out a faulty Android dependency if you add the following lines to your `android/app/build.gradle`
```
implementation("com.plaid.link:sdk-core:3.4.1") {
    force = true
}
```

**Comment by Crizzooo** [comment]
Thanks for the reply @amytang0 

What exactly goes in `yourAppHere` ? 

```
./gradlew :com.titannative:dependencies |
grep sdk-core

FAILURE: Build failed with an exception.

* What went wrong:
Project 'com.titannative' not found in root project 'titan-native'.
```

**Comment by zsweigart** [comment]
It's the name of your app level-module.  If you used the default project setup it will be `./gradlew :app:dependencies | grep sdk-core` and you will want to run this from the `/android` directory

**Comment by Crizzooo** [comment]
```
./gradlew :app:dependencies | grep sdk-cor
e
|    +--- com.plaid.link:sdk-core:3.4.0
|    +--- com.plaid.link:sdk-core:3.4.0
|    +--- com.plaid.link:sdk-core:3.4.0
|    +--- com.plaid.link:sdk-core:3.4.0
```

Will try the other suggestions and report back ASAP

**Comment by Crizzooo** [comment]
Thanks for the help @amytang0  @zsweigart 

After forcing 3.4.1, we are still hitting the above referenced `plaidlink://` URI Scheme Error

```
./gradlew :app:dependencies | grep sdk-core

+--- com.plaid.link:sdk-core:3.4.1
+--- com.plaid.link:sdk-core:3.4.1
+--- com.plaid.link:sdk-core:{strictly 3.4.1} -> 3.4.1 (c)
|    +--- com.plaid.link:sdk-core:3.4.0 -> 3.4.1
+--- com.plaid.link:sdk-core:3.4.1 (*)
+--- com.plaid.link:sdk-core:3.4.1
+--- com.plaid.link:sdk-core:{strictly 3.4.1} -> 3.4.1 (c)
|    +--- com.plaid.link:sdk-core:3.4.0 -> 3.4.1
+--- com.plaid.link:sdk-core:3.4.1 (*)
+--- com.plaid.link:sdk-core:3.4.1 (n)
+--- com.plaid.link:sdk-core:3.4.1
+--- com.plaid.link:sdk-core:{strictly 3.4.1} -> 3.4.1 (c)
|    +--- com.plaid.link:sdk-core:3.4.0 -> 3.4.1
+--- com.plaid.link:sdk-core:3.4.1 (*)
+--- com.plaid.link:sdk-core:3.4.1
+--- com.plaid.link:sdk-core:{strictly 3.4.1} -> 3.4.1 (c)
|    +--- com.plaid.link:sdk-core:3.4.0 -> 3.4.1
+--- com.plaid.link:sdk-core:3.4.1 (*)
```

Any ideas what could cause this WebView scheme error? 

At least with `7.0.7` I see Plaid stop spinning and show the beginning of the flow for a split second before this WebView error occurs. Can provide screen recording if that would help.

The error still does  not occur  in `variant=debug` mode.

**Comment by zsweigart** [comment]
Hi @Crizzooo
Have you been able to validate if our sample app works for you? One thing it could be if there is an issue with production builds but not debug builds is your proguard configuration.

**Comment by dtroupe-plaid** [comment]
Closing because Plaid Link React Native 7.x.x is no longer supported. If the issue persists in the latest SDK please open a new issue.
